src/Draggable/Sensors/MouseSensor/MouseSensor.js

```diff
@@ -123,7 +132,23 @@ export default class MouseSensor extends Sensor {
       return;
     }
 
-    const target = document.elementFromPoint(event.clientX, event.clientY);
+    const {document: currentHost} = event.view;
+
+    let {clientX, clientY} = event;
+
+    let target;
+
+    if (this.startedInsideIframe && (clientX < 0 || clientY < 0)) {
```

Comment:
`clientX` might be outside the iframe on the right side. You can check the upper limit using iframe's width or using elementFromPoint

```
const frameRect = frameElement.getBoundingClientRect()
const globalX = clientX + frameRect.left
const globalY = clientY + frameRect.top
const elementFromPoint = document.elementFromPoint(globalX, globalY)

return elementFromPoint === frameElement
```

This will work even if there are some elements displayed over the iframe element.